MatrixSDK/Space/MXSpaceService.swift

```diff
@@ -51,6 +58,9 @@ public class MXSpaceService: NSObject {
     private let processingQueue: DispatchQueue
     private let completionQueue: DispatchQueue
     
+    public private(set) var spaces: [MXSpace] = []
```

Comment:
Instead of a public property we should have like Android getters that returns space summaries, not directly `MXSpace`:
```
fun getSpaceSummaries(spaceSummaryQueryParams: SpaceSummaryQueryParams): List<RoomSummary>
fun getRootSpaceSummaries(): List<RoomSummary>
```
At the moment we do not have filtering but it will come later.